3-reveal/migrations/5-IRS/2-Zambia-2019/deploy/zambia_focus_area_irs.psql

```diff
@@ -54,11 +51,16 @@ SELECT * FROM
             WHEN irs_structures.foundStruct = 0 THEN 0
             ELSE CAST(irs_structures.sprayedStruct AS DECIMAL(7,2))/CAST(irs_structures.foundStruct AS DECIMAL(7,2))
         END AS spraySuccess,
+        (CAST(irs_structures.totStruct AS DECIMAL(7,2)) * 0.9) - CAST(irs_structures.sprayedStruct AS DECIMAL(7,2)) AS structures_remaining_to_90_se,
```

Comment:
is this affecting performance? do we need this value in the db? is that why we don't do this in the front-end?

Reply:
It doesn't significantly affect things.  We are doing it here because the general plan is for the web app to do as little processing as possible i.e. it just displays what its given (as much as we can get away with that).  This enables us to have the web app reports built off of pretty generic reusable code.

Reply:
For instance the calculations for IRS reports for Namibia and Zambia are different, but the web app doesn't really have to care

Reply:
OK I can be partially on board w/that, having the web app be configured can handle part of this goal as well, that said this looks good/makes sense